lib/bracket-matcher.coffee

```diff
@@ -54,11 +54,11 @@ class BracketMatcher
 
     skipOverExistingClosingBracket = false
     if @isClosingBracket(text) and nextCharacter is text and not hasEscapeCharacterBeforeCursor
-      if bracketMarker = _.find(@bracketMarkers, (marker) -> marker.isValid() and marker.getBufferRange().end.isEqual(cursorBufferPosition))
+      if (bracketMarker = _.find(@bracketMarkers, (marker) -> marker.isValid() and marker.getBufferRange().end.isEqual(cursorBufferPosition))) or @getScopedSetting("bracket-matcher.alwaysSkipClosingPairs")
```

Comment:
This conditional is getting a bit long - mind splitting it up into two lines?

Reply:
@50Wliu done